### DISCUSSION TITLE
变更系统字体后正常显示

### DISCUSSION THREAD

**Discussion by SaintChangeTheWorld** [martinet101/ElevenClock#1181] [General]
当我通过注册表或者第三方软件变更系统字体为其他字体时，我们的时间面板就无法显示了。
比如我在注册表中将微软雅黑的三个字体名字即注册表的键值换成阿里妈妈数黑体的字体名的时候，面板就无法显示了，当然软件也运行不起来，症状是任务栏里不会出现咱得图标，希望能优化支持下。

**Comment by marticliment** [comment]
Try setting a custom font on ElevenClock (ElevenClock Settings -> Clock appearance) before changing system font settings
